### ISSUE TITLE
Need for Speed™ Most Wanted (1262560)

### ISSUE THREAD

**Issue by KamilBaczkowski** [ValveSoftware/Proton#4510]
# Compatibility Report
- Name of the game with compatibility issues: Need for Speed™ Most Wanted
- Steam AppID of the game: 1262560

## System Information
- GPU: [AMD/ATI] Navi 10 [Radeon RX 5600 OEM/5600 XT / 5700/5700 XT]
- Driver/LLVM version: amdgpu
- Kernel version: 5.9.16
- Link to full system information report as [Gist](https://gist.github.com/KamilBaczkowski/35a310f334c29075fdc06574b18237fc):
- Proton version:
-- Proton Experimental
-- Proton 5.13-4
-- Proton 5.0-10

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1262560.log](https://github.com/ValveSoftware/Proton/files/5750543/steam-1262560.log)

## Symptoms
I can't properly install/launch Origin, and thus I cannot play the game. Sometimes the Origin installation goes through, but then Origin launcher wants to update. Currently, I can't even get it properly installed with Proton. After the installation starts, the window disappears after a while, the game just shows a _STOP_ button and nothing happens in the background. No new logs are outputted to the file.

## Reproduction
Try installing the game.

**Comment by kampfwuffi** [comment]
This is strange. Have you tried to force Proton 5.13-4 for this title in the games steam properties?
I'm running NFS MW on Manjaro with latest Kernel with Ryzen 3400G onboard graphics fine with ablolutely no problems and great fps.
Installation went fine too. I let the game install and press no button, even if there is no progress visible for some time.

- Have you checked if NFS MW is activated online at origin.com game library and Origin is linked to Steam?


The only issue: I need to force off all overlays to get a working wireless controller.

**Comment by KamilBaczkowski** [comment]
Yeah, the game is properly activated, I can play it all right on Windows. When I try to play on Linux, the installation hangs up in this state:
![image](https://user-images.githubusercontent.com/41474169/103467026-f5b30180-4d4a-11eb-9dcc-da8c2be159e5.png)
I can leave it like that even for an hour, and nothing happens.

**Comment by shyam-s00** [comment]
Yeah... This game works out of the box without any tweaks. It still plays well in 5.0-10 in my box without a problem. Give this 
[https://github.com/DrDoctor13/wine-origin-updater](url) custom origin updater a shot and see if you can get past the origin installation...

**Comment by KamilBaczkowski** [comment]
I re-installed the game and the installation went through with the usage of this script, but when launching the game, this happens:
![image](https://user-images.githubusercontent.com/41474169/104037222-b403da00-51d4-11eb-8429-496c7a243296.png). All the `Origin` windows are just transparent and I cant do anything

**Comment by KamilBaczkowski** [comment]
The game actually launched after a long while, but when I tried launching it again, all I can see are the origin transparent windows and one visible that says _Connecting to Origin…_.

**Comment by ghost** [comment]
I've been able to play properly, and I've seen those invisible windows when I used to use elementaryOS, but I don't see them on GNOME.
However, I haven't been able to play properly because since an origin update, it seems that the origin overlay forces the game to exit fullscreen mode, and since this game can never be set to fullscreen again, the game is currently unplayable (at least for me)
Sadly, the origin overlay can't be disabled in games that are bought through steam :/

**Comment by ghost** [comment]
So, the invisible windows now display, and they're windows of the origin client. And, the game still can't be fullscreened again if it goes to windowed mode, and it goes to window mode after starting it due to the origin overlay. This can be worked around by removing the preinstalled origin and replacing it with a normal origin installation

**Comment by parkerlreed** [comment]
Even with EA account linked to Steam, game stalls on loading screen no matter the Proton version currently.

**Comment by nezumisama** [comment]
I just tried to run the game today for the first time and with most Proton versions I tried (7-04, two versions of GE 7) the EA login window is blank so I don't even see the buttons or anything (I sort of can see them for a moment when I start clicking and dragging over it), so it's unusable. With proton experimental, the window works and I successfully registered an EA account connected with my steam one, and got the game window to open, but it stays on the origin spinner icon spinning forever (I gave it a few minutes). I tried reinstalling, but the same thing happens.

So the game is unplayable probably because of the EA DRM. Might've been a recent thing, because if you look at the protondb page, no one had such problems and it has a platinum status.

**Comment by parkerlreed** [comment]
They somehow deleted the English files so that's what causes it to constantly spin. You have to copy in the folder. 

https://steamcommunity.com/app/1262560/discussions/0/3461605794219302255/

**Comment by nezumisama** [comment]
You're right, it works now. So I successfully played a few minutes with proton experimental.

**Comment by Arborescent** [comment]
It works well on Proton 8.

To make racing wheels work, you can:
- Use the "Disable Steam Input" option
- Override the dinput8 library, make it as "Builtin (Wine)" on winecfg. You can run winecfg via protontricks

**Comment by ghost** [comment]
Hello, the game is not opening here. 
Using proton 8.0-5 rc, the game does not start.

Kubuntu, kde plasma, 23.10, wayland
Intel i3 1115G4
4GB RAM, 32GB swap
btrfs
ptbr utf8 abnt2
generic usb joystick

Thank you very much in advance

**Comment by ghost** [comment]
The super sampling mode of Need for speed most wanted 2012 closes the game, when activating the maximum mode, super sampling 4x. I think that's the name of the effect.
Also, the game did not save the progress in the Steam cloud, because when reinstalling the system, the game returns to the beginning.
The fps did not appear in the game either.
Also when minimized, the game does not return to full screen.

Kubuntu, kde plasma, 22.04
1920x1080p 60hz
Intel i3 1115G4
4GB RAM, 20GB swap
btrfs
ptbr utf8 abnt2
proton 8.0-5

Thanks

**Comment by simifor** [comment]
@AgoraSim21 The game doesn't crash for me hen using super sampling on an amd gpu, you might want to open an issue on the mesa issue tracker for your device. The game does not have steam cloud support, so file handling is likely handled by EA itself with their account linking.

**Comment by simifor** [comment]
I should also mention that the game exiting fullscreen when minimized is also its behavior in windows, so that isn't a proton issue. As you are on kde you might be able to force it to remain fullscreen even after that by using alt+f3→more actions→configure special application settings and then adding a rule to make the game fullscreen.

**Comment by ghost** [comment]
thanks 😄

**Comment by ghost** [comment]
This screen recently appeared here

![Screenshot_20240322_104609](https://github.com/ValveSoftware/Proton/assets/155112565/0b7da26d-7797-4afa-9f73-2b95406fbe0d)


The game wasn't opening, after I uninstalled burnout paradise remastered, need for speed most wanted 2012 didn't open anymore.
I can't say what happened 🤔🤔.

Phases:
install proton 7, and start need for speed
the screen above is shown.
Switch to proton 8, and start need for speed
the screen above is shown.
I also reinstalled burnout

👍

**Comment by simifor** [comment]
I don't know what's the trigger, but this seems to also happen on windows, you can try moving the licenses folder as indicated here and see if the error stops happening https://www.reddit.com/r/MECoOp/comments/189u76r/mass_effect_3_product_key_could_not_activate/

**Comment by ghost** [comment]
I can't run this game with proton experimental, 9 or 8. Trying 7 now.

**Comment by ghost** [comment]
Ok, for future readers:

- install with proton 7
- play with any other proton version (at the time of writing, proton experimental works, and so should proton 9 and 8)

**Comment by simifor** [comment]
@stereomato I tried installing the game with clean prefixes with proton 9 and proton experimental and the games launched fine (they launched themselves after EA launcher got installed). As the EA launcher was already linked to my steam account, the only thing I needed to install it was click ok when prompted, I don't know if this was your case as well or if you also needed to link your account when you installed the game.

**Comment by ghost** [comment]
for me the launcher would crash (seemingly) when using proton 9 or 8 or experimental. my account was linked ages ago

**Comment by Leandro599** [comment]
[steam-1262560.log](https://github.com/user-attachments/files/16972721/steam-1262560.log)

**Comment by Leandro599** [comment]
[steam-1262560.log](https://github.com/user-attachments/files/16995864/steam-1262560.log)
Debian 12

**Comment by Nj0be** [comment]
Using the steam deck, I can only run the game on first install. If I close and reopen the game, it shows a popup with "Preparing game..." text and after about 1 minute it crashes. I tried deleting the prefix numerous times, changing proton versions (7, 8, 9, experimental and latest proton ge), uninstalling and reinstalling the game. Until 2 days ago the game worked fine. I don't know how to resolve. Thanks

**Comment by hollov** [comment]
It works for me out of the box.

Although I want to use Gamescope because of fullscreen issue (as stated by some users on protondb), but then the game stucks at the animated loading screen. Not crashing, since animation and sound is playing. And this is only with Gamescope (without gs, it works)

**Comment by simifor** [comment]
@hollov I used proton 10/experimental, RX 9060xt, mesa 25.3.3 and gamescope 3.16.19, I can get ingame so I'm not getting stuck on the loading screen. What gamescope version are you using?

**Comment by hollov** [comment]
Hi @simifor ,
Thanks for getting back to me.

I'm in the middle of investigating this and it looks like it is because of my Hybrid setup: I'm on a laptop you see; intel + Nvidia 2080 Super hybrid GPU (MSI GS 75 Stealth). I'm investigating using AI and try to get help from the community as well; so any comment is much appreciated.

AI says (based on the logs) that I'm stuck because iGPU is used (but it's AI; maybe it's wrong)
When I try to force dGPU then KWin crashes and restarts (hence every app is killed as well, including the game) ;(

Using Nobara 43 and everything is up to date, so I'm pretty sure I'm on gamescope 3.16.19 as well (cannot check it atm)
